packages/binaryninja-free.vm/binaryninja-free.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>binaryninja-free.vm</id>
+    <version>4.2</version>
```

It seems the version is `4.2.6455`:

```suggestion
    <version>4.2.6455</version>
```